src/main/kotlin/io/sledge/deployer/crx/command/HttpMethodWrapper.kt

```diff
@@ -0,0 +1,52 @@
+package io.sledge.deployer.crx.command
+
+import com.github.ajalt.clikt.output.TermUi
+import io.sledge.deployer.common.Retry
+import io.sledge.deployer.exception.SledgeCommandException
+import io.sledge.deployer.http.HttpClient
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.runBlocking
+import okhttp3.Response
+
+fun executePost(httpClient: HttpClient, command: Command, packageName: String, parameter: Pair<String, *> = Pair("", "")) {
+    runBlocking {
+        delay(2000)
+        Retry().retry {
```

Comment:
Where do we pass the retries option from the cli if provided? I think this is still missing right?